.github/workflows/docker-build-test.yaml

```diff
@@ -135,7 +135,7 @@ jobs:
 
   rust-images-performance:
     needs: [permission-check, determine-docker-build-metadata]
-    uses: aptos-labs/aptos-core/.github/workflows/workflow-run-docker-rust-build.yaml@main
+    uses: aptos-labs/aptos-core/.github/workflows/workflow-run-docker-rust-build.yaml
```

Comment:
try changing this to `./.github/workflows/workflow-run-docker-rust-build.yaml` to canary on this PR

Reply:
still run on high-perf-docker not the experimental-docker